### PR TITLE
Design and implement the functionalities for "add process" page in Process center

### PR DESCRIPTION
**Pull request by isurusuranga**

### CODE REVIEW

File: modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs [hunk 1]
```diff
@@ -34,10 +39,11 @@
...
                 type: 'POST',
-                data: { 'processName': $("#pName").val(), 'processVersion' : $("#pVersion").val(), 'processOwner' : $("#pOwner").val(), 'processTags' : $("#pTags").val()},
+                data: { 'processInfo': getProcessInfo() },
                 success: function (response) {
                     $("#processTextOverviewLink").attr("href", "../../assets/process/details/" + response);
                     $("#bpmnOverviewLink").attr("href", "../../assets/process/details/" + response);
+                    window.location = "../../assets/process/list";
                 },
                 error: function () {
                     alert("error");
```
Comment (vinok88) [inline modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs]: can we include error info here?

Comment (isurusuranga) [inline modules/extensions/process/publisher/process/themes/default/partials/create_asset.hbs]: Already fixed

File: modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java [hunk 2]
```diff
@@ -143,6 +151,24 @@ public String createProcess(String processName, String processVersion, String pr
                 appendText(doc, propertiesElement, "bpmnpath", mns, "NA");
                 appendText(doc, propertiesElement, "bpmnid", mns, "NA");
 
+                if(successor.length() != 0){
+                    for(int i = 0 ; i < successor.length() ; i++){
```
Comment (isudana) [inline modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java]: Code formatting need to be corrected according to wso2 code style template

Comment (isurusuranga) [inline modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java]: Already fixed.

File: modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java [hunk 3]
```diff
@@ -143,6 +151,24 @@ public String createProcess(String processName, String processVersion, String pr
...
+                        Element successorElement = append(doc, rootElement, "successor", mns);
+                        appendText(doc, successorElement, "name", mns, successor.getJSONObject(i).getString("name"));
+                        appendText(doc, successorElement, "path", mns, successor.getJSONObject(i).getString("path"));
+                        appendText(doc, successorElement, "id", mns, successor.getJSONObject(i).getString("id"));
+                    }
+                }
+
+                if(predecessor.length() != 0){
```
Comment (Kishanthan) [inline modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java]: Formatting issue.

Comment (isurusuranga) [inline modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/ProcessStore.java]: Already fixed in https://github.com/wso2/product-pc/pull/36 (See under git comment called "Reformat the code")

**Pull request merged**